xrview/xrechnung.cpp

```diff
@@ -101,11 +101,8 @@ void XRechnung::slotErrorOccurred(QProcess::ProcessError error)
     }
 }
 
-void XRechnung::slotDomFinished(int exitCode, QProcess::ExitStatus exitStatus)
+void XRechnung::slotDomFinished(int /*exitCode*/, QProcess::ExitStatus /*exitStatus*/)
 {
-    Q_UNUSED(exitStatus)
-    Q_UNUSED(exitCode)
```

Comment:
I'd rather implement a check and at least give a debug message if the call failed.

Reply:
The standard error is printed out unconditionally. Also, we are connected to the errorOccurred signal. 

Should we maybe print out other errors than `FailedToStart` in slotErrorOccurred?